src/main/java/me/totalfreedom/totalfreedommod/rank/RankManager.java

```diff
@@ -152,13 +158,13 @@ public void onPlayerJoin(PlayerJoinEvent event)
             player.getInventory().clear();
             player.setOp(false);
             player.setGameMode(GameMode.SURVIVAL);
-            plugin.pl.getPlayer(player).getFreezeData().setFrozen(true);
+            plugin.pl.getPlayer(player).setEditBlocked(true);
             player.sendMessage(ChatColor.RED + "You are marked as an impostor, please verify yourself!");
             return;
         }
 
         // Set display
-        if (isAdmin || FUtil.DEVELOPERS.contains(player.getName()))
+        if (isAdmin || FUtil.DEVELOPERS.contains(player.getName()) || FUtil.FOUNDER.contains(player.getName()))
```

Comment:
Okay, what I am getting at is why are you treating the founders login differently to an owners login? I would understand this if it was developers || owners already, but its not, its just developers. Treat founder login messages the same way an owner is, and I think thats through a custom login message in the admin files.